examples/nlp/README.md

```diff
@@ -0,0 +1,47 @@
+# Implementation of NLP algorithm using Zenml
+
```

Comment:
We have recently refactored all our example README files. Would you mind adopting a similar style to our other READMEs? 
Additionally it would be great to have some context regarding the model and what it does, so that beginners in the NLP space can quickly grasp what the pipeline is all about.

Reply:
Additionally for our examples we usually have the code at the same level as the README,  could you also please remove the `token-classification`-subdirectory?

Reply:
Actually, I am planning to add more NLP examples i.e text classification, question answering etc, So I made a sub-dir. You suggest me what should I do?